iron-request.html

```diff
@@ -286,9 +286,16 @@
           handleAs = 'text';
         }
 
-        // In IE, `xhr.responseType` is an empty string when the response
-        // returns. Hence, caching it as `xhr._responseType`.
-        xhr.responseType = xhr._responseType = handleAs;
+        //placing the following line inside try catch as some browsers like UC doesn't support JSON as response type. It throws SYNTAX_ERROR: DOM EXCEPTION 12
+        try{
```

Comment:
I'm not comfortable running a try-catch on every `.send`.

Can you wrap this Polymer call in an IIFE, check this try-catch (so it only runs once), create a boolean on whether to skip this or not, and skip this with an `if` statement?